threadpool/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Lets remove this CMake file and move its content to the top level CMake.

Reply:
Since this CMake file is used only for this branch lets keep it as it is. However, when we merge this branch with the master branch we will have to move the content of this CMake file to the CMake file in the master branch.